integration/resource_lacework_integration_gar_test.go

```diff
@@ -28,11 +28,11 @@ func TestIntegrationGARCreate(t *testing.T) {
 		})
 		defer terraform.Destroy(t, terraformOptions)
 
-		// Create new Email Alert Channel
+		// Create new Google Artifact Registry
 		create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
 		assert.Equal(t, "Google Artifact Registry Example", GetIntegrationName(create))
 
-		// Update Email Alert Channel
+		// Update Google Artifact Registry
```

Comment:
THANK YOU!!! 

![tenor-163093302](https://user-images.githubusercontent.com/5712253/129906520-04fd09ea-e8ba-4370-8b95-9a1da818b71e.gif)